### DISCUSSION TITLE
下载数据（后端下载）

### DISCUSSION THREAD

**Discussion by Baili-BL** [antvis/S2#2667] [General]
大佬，求解后端下载如何能跟前端下载的内容格式保持一致，模拟前端的数据
https://s2.antv.antgroup.com/manual/advanced/analysis/export

**Comment by lijinke666** [comment]
需要后端参照前端源码的实现, 使用后端的编程语言实现一遍

https://github.com/antvis/S2/blob/de7c97b862e5b467fd335dd65f9dac5a95e4b621/packages/s2-core/src/utils/export/copy/core.ts

**Reply by Baili-BL** [reply]
感谢~

**Reply by Baili-BL** [reply]
大佬，补充问一下：我理解是不是通过chrome driver 去驱动前端代码，去调取前端代码实现。这样才能保持前后端的图表都一致？
